manage/scheduler.rst

```diff
@@ -66,7 +88,7 @@ Disable notifications
 Note
    You can use the note field to describe the purpose of the scheduler.
    This is only visible to other admins when they are editing the scheduler.
-   It is *not* a way to add notes to tickets.
+   It is *not* a way to add notes to objects.
```

Comment:
Again, the note action is exclusive to the ticket object, so perhaps we can leave this distinction in place.

Reply:
This note field is present for any object selection and should be used for comments on the purpose of the scheduler. So I think it should be fine, or am I wrong?

Reply:
Whoops, you're absolutely right, I had Note action in my mind, not Note field. Carry on, nothing to see here!